src/pages/OfferNew/Compare/CompareTable.tsx

```diff
@@ -0,0 +1,314 @@
+    outline: none;
+  }
+
+  svg {
+    width: 6px;
+  }
+`
+
+const PrimaryCompanySection = styled('div')`
+  width: 100%;
+  max-width: 178px;
+  height: 100%;
+  padding: 25px;
+  box-sizing: border-box;
+  background-color: ${colorsV2.lightgray};
+  position: relative;
+
+  :before,
+  :after {
+    content: '';
+    position: absolute;
+    left: 0;
+    background-color: ${colorsV2.lightgray};
+    width: 100%;
+    height: 10px;
+  }
+
+  :before {
+    top: -10px;
+    border-top-left-radius: 8px;
+    border-top-right-radius: 8px;
+  }
+  :after {
+    bottom: -10px;
+    border-bottom-left-radius: 8px;
+    border-bottom-right-radius: 8px;
+  }
+`
+
+const PrimaryCompanyHead = styled(ColumnHead)`
+  display: flex;
+  justify-content: center;
+`
+
+const CompanyColumnRow = styled(ColumnRow)`
+  font-size: 15px;
+  justify-content: center;
+  text-align: center;
+  letter-spacing: -0.2px;
+`
+
+const OtherCompaniesSection = styled('div')`
+  width: 100%;
+  max-width: 178px;
+  height: 100%;
+  padding: 25px 16px;
+  box-sizing: border-box;
+  position: relative;
+`
+
+interface OtherCompanyHeadProps {
+  currentCompany: CompanyProperties | null
+  dropdownIsVisible: boolean
+}
+
+const OtherCompanyHead = styled('button')<OtherCompanyHeadProps>`
+  width: 100%;
+  display: flex;
+  height: 25px;
+  align-items: center;
+  justify-content: space-between;
+  border: none;
+  margin-bottom: 58px;
+  font-size: 16px;
+  font-weight: 500;
+  color: ${(props) =>
+    hexToRgba(
+      colorsV2.black,
+      props.currentCompany !== null && !props.dropdownIsVisible ? 1 : 0.2,
+    )};
+  transition: all 0.1s ease;
+  cursor: pointer;
+
+  :focus {
+    outline: none;
+  }
+
+  > svg {
+    width: 14px;
+    transition: all 0.1s ease;
+    ${(props) => props.dropdownIsVisible && `transform: rotate(180deg);`}
+    ${(props) => props.dropdownIsVisible && `fill: ${colorsV2.violet500};`}
+  }
+
+  :hover {
+    ${(props) => !props.dropdownIsVisible && `color: ${colorsV2.black};`}
+
+    svg {
+      fill: ${colorsV2.violet500};
+    }
+  }
+`
+
+const Dropdown = styled('div')<{ visible: boolean }>`
+  background: ${colorsV2.white};
+  width: 100%;
+  position: absolute;
+  height: calc(100% - 61px);
+  left: 0;
+  top: 60px;
+  transition: all 0.2s;
+  opacity: ${(props) => (props.visible ? 0.9 : 0)};
+  visibility: ${(props) => (props.visible ? 'visible' : 'hidden')};
+  padding: 16px;
+  box-sizing: border-box;
+  border-top: 1px solid ${colorsV2.lightgray};
+`
+
+const DropdownRow = styled('button')`
+  width: 100%;
+  background: none;
+  border: none;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: left;
+  color: ${colorsV2.darkgray};
+  margin-bottom: 14px;
+  cursor: pointer;
+
+  :focus {
+    outline: none;
+  }
+
+  :hover {
+    color: ${colorsV2.violet500};
+  }
+`
+
+const getProperty = (key: string, value: any): any => {
+  if (key === 'deductible') {
+    return <ColumnRowPrimaryContent>{value} kr</ColumnRowPrimaryContent>
```

Comment:
could we utilize the `Intl` api instead? like so https://github.com/HedvigInsurance/hedgehog/blob/master/src/lib/intl.ts#L7-L10

Reply:
Yeah maybe we could, but I think it's better to postpone this to the point when we get real data